### ISSUE TITLE
The '~' glyph it's not visually balanced

### ISSUE THREAD

**Issue by gleesik** [microsoft/cascadia-code#237]
# Description of the new feature/enhancement

The '~' glyph it's not visually balanced. Sometimes when I use it in programming it can be confusing and not visually pleasing. It looks like it's rotated (in the wrong direction) because the geometric weight center isn't horizontally centered. I find this to be especially true when I use this glyph in the context of a file path.

This image shows a context where the problem can be identified:

![image](https://user-images.githubusercontent.com/22911281/74031040-2fec1580-49b9-11ea-8522-ce65150a5af8.png)

I think this is happening because some parts of the glyph are not horizontally centered. The image below shows a more articulate view of the issue:

![image](https://user-images.githubusercontent.com/22911281/74031446-316a0d80-49ba-11ea-873a-6fc282e3391b.png)

__OBS__: The section of the glyph from the 1 area _is bigger_ than the section of the glyph from the 2 area. This makes the glyph to look unbalanced.

# Proposed technical implementation details
I think that if the glyph will be redesigned to be more even horizontally it will look better and it will not create unbalanced perception of the visual information.
Here is an image with the same version of the glyph, but rotated a little bit (15deg):

![image](https://user-images.githubusercontent.com/22911281/74032037-ae49b700-49bb-11ea-8190-59003041032e.png)

__OBS__: The rotated version is not perfect, but it can help to identify the issue.

**Comment by madig** [comment]
I have found that when rotating tildes like in your proposal, they sometimes are less distinguishable at smaller sizes. The current one sticks out, which I find helpful here and there.

**Comment by gleesik** [comment]
@madig you might be right about the fact that at smaller sizes a simple rotation can make it less distinguishable. My proposal is not to rotate it, but to make it _horizontally even_. Rotating it's just a very simple way of achieving that __horizontal balance__.

__OBS__: I think the horizontal balance can be achieved by other techniques than just a simple rotation. In fact a simple rotation it's not necessarily in harmony with the design 'personality' of the typeface.

**Comment by Fallyx** [comment]
Is it possible to have a stylistic set with an alternative tilde?

**Comment by gleesik** [comment]
@Fallyx That might be _a solution_ that can work for many people and it should be considered. I think it can be very nice to have something like that. However a solution like this will need additional maintability effort that should be taken into consideration.

**OBS1:** I think that if the main font from the typeface will adjust the horizontal balance of the tilde glyph will benefit everyone. It shouldn't be completely changed, but it can be adjusted a bit.

**OBS2:** For people with **OCD** (a.k.a. Obsessive Compulsive Disorder) or **ASD** (a.k.a. Autism Spectrum Disorder) _unbalanced glyphs_ can cause **anxiety**. Also many people that write code have one or both mentioned disorders or other _similar derived disorders_ (possibly not even diagnosed or aware of this). I think this is an argument that is worth to be taken into consideration.

**Comment by aaronbell** [comment]
Thanks for the feedback! I definitely think that there can be some improvement on the tilde treatment as a whole—if you look at bug #216, I noted that I've treated the standalone versions differently than the diacritical version (which annoys the heck out of me). I want to go through and bring alignment to the whole lot of them. 

Other things have kept me from addressing this sooner, but yes, this is on the list of fixes :). 

Closing as a dupe of #216
